### PR TITLE
Set correct property for non-conformant mgmt projects

### PR DESCRIPTION
**Pull request by weshaggard**

### CODE REVIEW

File: eng/pipelines/mgmt.yml [hunk 1]
```diff
@@ -6,7 +6,10 @@ pr:
     - '*-preview'
   paths:
     include:
+    - mgmt.proj
```
Comment (markcowl) [inline eng/pipelines/mgmt.yml:9]: Do we want to include the top level Directory.Build.props / targets in here as well?

File: eng/scripts/Update-Mgmt-Yml.ps1 [hunk 2]
```diff
@@ -38,7 +38,11 @@ $PrIncludes = New-Object "System.Collections.Generic.List[String]"
 
 $MgmtDirs = Get-ChildItem -Path "$PackagesPath" -Directory -Recurse -Depth 1 | Where-Object { $_.FullName -match "(.Microsoft\.Azure(Stack)?\.Management.)|(.\\mgmt)" }
 
+$Includes.Add('mgmt.proj')
 $Includes.Add('eng/pipelines/mgmt.yml')
+$Includes.Add('eng/mgmt')
+$Includes.Add('sdk/mgmtcommon')
+
```
Comment (markcowl) [inline eng/scripts/Update-Mgmt-Yml.ps1:45]: same question

File: sdk/azurestack/Directory.Build.props [hunk 3]
```diff
@@ -0,0 +1,6 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <IsLegacyMgmtProject Condition="'$(IsLegacyMgmtProject)' == ''">true</IsLegacyMgmtProject>
```
Comment (markcowl) [inline sdk/azurestack/Directory.Build.props:3]: I wonder if we want to insert this in all the management csproj files.  I am wondering if we are disabling any build scenarios, like building from the local directory for management projects.

**Review by markcowl — CHANGES_REQUESTED** [review]

File: eng/pipelines/mgmt.yml (hunk 1 above)
Comment (weshaggard) [reply eng/pipelines/mgmt.yml:9]: Yeah I guess we could include those as well I guess we just have to figure out how far we want to go with the list.

File: sdk/azurestack/Directory.Build.props (hunk 3 above)
Comment (weshaggard) [reply sdk/azurestack/Directory.Build.props:3]: You could manually put them in all project files but I think that is overkill. The way we have it setup now this should work for building these projects in any context. Even if you build from inside the project directory this should still get imported correctly.

File: eng/pipelines/mgmt.yml (hunk 1 above)
Comment (weshaggard) [reply eng/pipelines/mgmt.yml:9]: I added in my follow-up PR.

**Pull request merged**